### PR TITLE
chore(CHANGELOG): update to v4.4.0

### PR DESCRIPTION
**Pull request by nikku**
Updates the CHANGELOG to v4.4.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
 ___Note:__ Yet to be released changes appear here._
 
+## 4.4.0
+
+### General
+
+* `FEAT`: enable `Backspace` on MacOS for element removal ([#1989](https://github.com/camunda/camunda-modeler/issues/1989))
+* `FEAT`: enable `SHIFT + click` for multi-selection ([#1964](https://github.com/camunda/camunda-modeler/issues/1964))
+* `FIX`: correct switching between diagram and XML ([#1925](https://github.com/camunda/camunda-modeler/issues/1925))
+* `CHORE`: capture deployment and process variables in telemetry events
```
Comment (MaxTru) [inline CHANGELOG.md:16]: Is there a reason we don't reference https://github.com/camunda/camunda-modeler/issues/1971 here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+
+* `FIX`: correct removal of element templates from events ([#1990](https://github.com/camunda/camunda-modeler/issues/1990))
+* `FIX`: unlink incompatible element template during replace ([#1961](https://github.com/camunda/camunda-modeler/issues/1961))
 * `FIX`: store variable events for conditional events in the correct property ([#836](https://github.com/camunda/camunda-modeler/issues/836))
 * `FIX`: re-enable `entriesVisible` property of templates ([#1975](https://github.com/camunda/camunda-modeler/issues/1975))
+* `FIX`: correct label of start instance tool ([#1777](https://github.com/camunda/camunda-modeler/issues/1777))
+
+### DMN
+
+* `FEAT`: allow decision table name to take empty space
```
Comment (MaxTru) [inline CHANGELOG.md:28]: Any reason we don't reference https://github.com/bpmn-io/dmn-js/issues/579 here?
(It is not in camunda-modeler, but we also did that in the past?)

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: focus cell in newly added row ([#928](https://github.com/camunda/camunda-modeler/issues/928))
+* `FIX`: render hit policy drop down in the correct location
```
Comment (MaxTru) [inline CHANGELOG.md:30]: Same as before -> https://github.com/bpmn-io/dmn-js/issues/589

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: enable `SHIFT + click` for multi-selection ([#1964](https://github.com/camunda/camunda-modeler/issues/1964))
+* `FIX`: correct switching between diagram and XML ([#1925](https://github.com/camunda/camunda-modeler/issues/1925))
+* `CHORE`: capture deployment and process variables in telemetry events
+
+### BPMN
+
+* `FIX`: correct removal of element templates from events ([#1990](https://github.com/camunda/camunda-modeler/issues/1990))
+* `FIX`: unlink incompatible element template during replace ([#1961](https://github.com/camunda/camunda-modeler/issues/1961))
 * `FIX`: store variable events for conditional events in the correct property ([#836](https://github.com/camunda/camunda-modeler/issues/836))
 * `FIX`: re-enable `entriesVisible` property of templates ([#1975](https://github.com/camunda/camunda-modeler/issues/1975))
```
Comment (MaxTru) [inline CHANGELOG.md:23]: Why not name them `element templates` here to be precise?

File: CHANGELOG.md (hunk 2 above)
Comment (nikku) [reply CHANGELOG.md:28]: Sorry for that, will add the library version bumps. 

In the past, we did not reference these library issue. Instead we simply stated what was changed and referred to the actual library release.

File: CHANGELOG.md (hunk 4 above)
Comment (nikku) [reply CHANGELOG.md:23]: Fix incoming.

File: CHANGELOG.md (hunk 3 above)
Comment (nikku) [reply CHANGELOG.md:30]: I'd not cross-reference issues across project boundaries. Instead we clearly mention the new bpmn-js version contained, thus giving experts a chance to dive into the actual changes.

File: CHANGELOG.md (hunk 1 above)
Comment (nikku) [reply CHANGELOG.md:16]: Just did not want to mention more internals for already internal things.

**Comment by nikku** [comment]
Updated the Changelog, added the library upgrades as well as `templates -> element templates`.

File: CHANGELOG.md (hunk 1 above)
Comment (MaxTru) [reply CHANGELOG.md:16]: :+1:

**Review by MaxTru — APPROVED** [review]

**Pull request merged**